src/app/assets/stylesheets/app.scss

```diff
@@ -138,3 +138,22 @@ a.ptc {
 h1, h2, h3, h4, h5, h6 {
   line-height: 1.2 !important;
 }
+
+.phishy {
+  display: inline;
+  color: red;
+}
+
+a[rel="noopener"]:after {
```

Comment:
I would rather insert real image after a link - this way you can add hover message saying "External link", also I think we can make it a little bit smaller and position better if use separate img tag.

Reply:
@relativityboy why did these css changes disappear when you did a master merge? strange!